examples/ransomware_detection/run.py

```diff
@@ -137,40 +134,29 @@ def run_pipeline(debug,
     # Load ransomware detection configuration.
     rwd_conf = load_yaml(conf_file)
 
-    # Exclude columns that are not required.
-    cols_exclude = ["SHA256"]
-
     # Only intrested plugins files will be read from Appshield snapshots.
-    interested_plugins = ['ldrmodules', 'threadlist', 'envars', 'vadinfo', 'handles']
+    interested_plugins = rwd_conf["intrested_plugins"]
+
+    # Columns from the above intrested plugins.
+    plugins_schema = rwd_conf['plugins_schema']
 
     # Columns from the above intrested plugins.
     cols_interested_plugins = rwd_conf['raw_columns']
 
     # Feature columns used by the model.
-    model_features = rwd_conf['model_features']
-
-    # Features to include in the DF, superset of model_features along with a few that the model doesn't receive
-    feature_columns = model_features + rwd_conf['features']
+    feature_columns = rwd_conf['model_features']
 
     # File extensions.
     file_extns = rwd_conf['file_extensions']
 
     # Set source stage.
     # This stage reads raw data from the required plugins and merge all the plugins data into a single dataframe
     # for a given source.
-    pipeline.set_source(
-        AppShieldSourceStage(
-            config,
-            input_glob,
-            interested_plugins,
-            cols_interested_plugins,
-            cols_exclude=cols_exclude,
-            watch_directory=watch_directory,
-        ))
+    pipeline.set_source(KafkaSourceStage(config, bootstrap_servers=bootstrap_servers, group_id=group_id, input_topic=input_topic))
 
-    # Add a monitor stage.
-    # This stage logs the metrics (msg/sec) from the above stage.
-    pipeline.add_stage(MonitorStage(config, description="FromFile rate"))
+    # Add a appshield partitioner stage.
+    # This stage combine the plugins together to combined dataframe
+    pipeline.add_stage(AppshieldPartitionerStage(config, interested_plugins, cols_interested_plugins, plugins_schema))
```

Comment:
This changes the example source from using a file to using Kafka. This adds a layer of complexity that didnt exist before. What was the main reason for using Kafka instead of a file as the source?